modules/payloads/singles/php/meterpreter_reverse_tcp.rb

```diff
@@ -39,6 +39,17 @@ def generate
     session_guid = '\x00' * 16
     met = met.sub(%q|"SESSION_GUID", ""|, %Q|"SESSION_GUID", "#{session_guid}"|)
 
+    ds = datastore
+    ds ||= framework.datastore
+
+    if (ds['MeterpreterDebugBuild'])
+      payload.gsub!("define(\"DEBUGGING\", false);", "define(\"DEBUGGING\", true);")
+
+      if (ds['MeterpreterDebugLogging'])
+        payload.gsub!('define("DEBUG_LOGFILE_PATH", "");', "define(\"DEBUG_LOGFILE_PATH\", \"#{ds['MeterpreterDebugLogging'].split('rpath:').last}\");")
```

Comment:
Same feedback as before; It might make sense to extract this parsing logic into a shared place so folk don't have to copy/paste the same logic across payloads